python/pipx/Portfile

```diff
@@ -20,7 +20,21 @@ checksums           rmd160  64b198d937c3c7b12531a81828c7b813fa9a597a \
 
 python.default_version 37
 python.versions     36 37 38
```

Comment:
the two lines above should be removed if you're going to use ```variants``` (the last one wasn't anyway not doing anything anyways). You likely want to set a default version if no variant is provided - there are many Portfiles that do similar things so you can take a look.

Reply:
Yes, now that those became variants these two lines can go away and you should set the default variant. I think we could set the default to `38`, but I prefer small commits that do only one thing, so keep `37` as the default variant in this commit. If you want you can another one to change to `38` or I can do that myself afterwards.

Reply:
I think when I made the edits, I looked through other ports and decided that `default_version` would be better than a default variant for ergonomic reasons.

I'll submit a patch for 38 after this one lands.